src/index.js

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
+import 'bootstrap/dist/css/bootstrap.css';
+import "jquery/dist/jquery.min.js";
+import "popper.js/dist/umd/popper.min.js";
+import "bootstrap/dist/js/bootstrap.min.js";
+import './index.css';
+
```

Comment:
I am still not sure if bootstrap is the perfect third party library to use here. I would recommend you to write the css from scratch. Would be really great if you can move ahead with SASS.